Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -72,6 +72,13 @@ def norm(self, **kwargs):
     def set_norm(self, norm=None):
         '''Sets the norm of the operator to a custom value.
         '''
+        try:

```

Comment:
Maybe better is to use `from numbers import Number`
and raise ValueError with a different msg a) if negative b) if not a Number

Reply:
thanks Vagellis - have changed this